pkg/volume/flexvolume/mounter.go

```diff
@@ -93,7 +94,7 @@ func (f *flexVolumeMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs)
 
 	if !f.readOnly {
 		if f.plugin.capabilities.FSGroup {
-			volume.SetVolumeOwnership(f, mounterArgs.FsGroup, mounterArgs.FSGroupChangePolicy)
+			volume.SetVolumeOwnership(f, mounterArgs.FsGroup, mounterArgs.FSGroupChangePolicy, util.FSGroupCompleteHook(f.plugin.GetPluginName()))
```

Comment:
Should the plugin name be "flexvolume" or the name of the flex driver?

Reply:
I change it to "kubernetes.io/flexvolume:[flex driver name]". see https://github.com/kubernetes/kubernetes/pull/95892/.